src/docs/releases/2.1.0.md

```diff
@@ -63,6 +63,17 @@ In the Roles feature, there were previously `AssignRoles` and `AssignRole_{RoleN
 !!! warning
     Please review all your recipes and replace occurrences of `AssignRoles` with `AssignRoleToUsers`, and `AssignRole_{RoleName}` with `AssignRoleToUsers_{RoleName}`.
 
+#### Roles Now Include Types
+
+A new `Type` property has been added to the `IRole` interface, which allows defining a role's behavior:
+
+- **`Standard`**: A typical role that allows granular permission assignment.
+- **`System`**: Non-removable system roles such as 'Anonymous' and 'Authenticated.'
+- **`Owner`**: A special role type that automatically grants all permissions without needing to specify them. This type is designed for roles like `Administrator` or `SiteOwner`.
```

Comment:
Yes I want to do that is a separate PR as this would touch multiple file.

Reply:
I suggest doing it here, because this PR is only half of the point of the corresponding issue.

Reply:
deprecating it alone will generate warning. I want it in a separate PR because we have to also create new permissions where `SiteOwner` was used. so it's not just deprecating it

Reply:
Yeah, I understand, but all of this is tightly related to the changes in this PR.

Reply:
okay. deprecated it here

Reply:
Thank you!